src/server/route.go

```diff
@@ -46,10 +49,10 @@ func registerRoutes() {
 	beego.Router("/api/internal/switchquota", &api.InternalAPI{}, "put:SwitchQuota")
 	beego.Router("/api/internal/syncquota", &api.InternalAPI{}, "post:SyncQuota")
 
-	beego.Router("/service/notifications/jobs/adminjob/:id([0-9]+)", &admin.Handler{}, "post:HandleAdminJob")
+	router.NewRoute().Method(http.MethodPost).Path("/service/notifications/jobs/adminjob/:id([0-9]+)").HandlerFunc(ok)
```

Comment:
There is a handler for handling all kinds of jobs related webhook: https://github.com/goharbor/harbor/blob/master/src/server/route.go#L55